src/Modules/Build.lua

```diff
@@ -937,7 +937,7 @@ function buildMode:SyncLoadouts(reset)
 				if linkIdentifier then
 					for linkId in string.gmatch(linkIdentifier, "[^%,]+") do
 						transferTable["setId"] = set
-						transferTable["setName"] = string.match(setTitle, "(.+)% {")
+						transferTable["setName"] = string.match(setTitle, "(.+)% {") or "Default"
```

Comment:
```suggestion
						transferTable["setName"] = string.match(setTitle, "(.+)% *{")
```

This was going to be my fix, but yours works too.

Reply:
I actually did that at first but it doesnt work because string.match still returns nil if the string is just empty